packages/dev-utils/src/devApp/render.tsx

```diff
@@ -141,7 +146,9 @@ export class DevAppBuilder {
       this.defaultPage = path;
     }
 
-    if (opts.title) {
+    if (opts.sidebarItem) {
+      this.sidebarItems.push(opts.sidebarItem);
```

Comment:
I'm thinking it would be best to separate this one out into `.addSidebarItem(...)` instead? Thinking there might be cases where you want to add items without a page, so nicer to not have them too coupled.